spec/models/green_lanes/category_assessment_spec.rb

```diff
@@ -307,14 +310,18 @@
 
     it { is_expected.to include gl_measure }
 
+    # rubocop:disable RSpec::EmptyExampleGroup
```

Comment:
I wonder if we can just extend this cop/configure it to support it_with_refresh, etc